media-library/overview/upload-files.md

```diff
@@ -52,3 +52,41 @@ You can share a folder with a restricted user by providing "Can contribute" or "
 ## Using the Upload API
 
 You can also upload images to ImageKit.io's media library programmatically. Read the Upload API documentation [here](../../api-reference/upload-file-api/).
+
+## Upload settings
+
+* Extensions
+
+Extensions let you perform certain advanced operations on your assets during upload. For reference about extensions, [read here](../../extensions/overview).
+
+* Custom focus area
+
+Define an important area in the image during upload. This is an optional field and is only relevant for image type files, [read here](./custom-focus-area.md).
```

Comment:
> image type files, [read here]

This sentence looks a little malformed. It would be better to add a separate sentence like "To learn more about X, read here." like in the extensions section above.

Same for all sections here that have a `read more` link